backend/src/main/java/com/cruru/config/ClockConfig.java

```diff
@@ -0,0 +1,14 @@
+package com.cruru.config;
+
+import java.time.Clock;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ClockConfig {
+
+    @Bean
+    public Clock clock() {
+        return Clock.systemDefaultZone();
```

Comment:
```suggestion
        return Clock.system(ZoneId.of("Asia/Seoul"));
```
시간대를 서울로 고정시켜놓는건 어떤가요? 현재 DB도 TZ가 서울로 되어있는 것으로 알고 있습니다.